bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/Journal.java

```diff
@@ -499,6 +502,10 @@ public void run() {
                     journalStats.getForceWriteGroupingCountStats()
                             .registerSuccessfulValue(numReqInLastForceWrite);
 
+                    if (requestProcessor != null) {
```

Comment:
Here we trigger all channels to flush `pendingSendResponses`, shall we only pick the requestHandler which is written to the logFile to flush `pendingSendResponses`, it may reduce the unnecessary iteration in the loop.
 
```
    public void flushPendingResponses() {
        for (Channel c : allChannels) {
            c.pipeline().fireUserEventTriggered(BookieRequestHandler.EVENT_FLUSH_ALL_PENDING_RESPONSES);
        }
    }
```
If there are 1000 channels, maybe only 2 channels need to flush `pendingSendResponses`